source/contributing/implementation/debugging/ide.rst

```diff
@@ -2,26 +2,91 @@
-- If IntelliJ is already open, click on ``File``, followed by ``Open Project``.
+Ensure the Gradle plugin is enabled. To do so, go to ``File`` -> ``Settings`` -> ``Plugins``. Look for the ``Gradle`` 
+plugin in the ``Installed`` tab and verify a checkmark next to the plugin's name. If you do not see the Gradle plugin, 
+search for it in the ``Marketplace`` tab and install it.
 
-Navigate to the implementation's root directory and select ``build.gradle``. Click OK. IntelliJ loads your workspace 
-and builds the project. You can begin debugging the code.
+Project SDK
+-----------
 
+The project SDK should be ``1.8.0_20`` or higher. The project language level should be ``8 - Lambdas, type annotations 
+etc.``
 
-Eclipse
--------
+Run/Debug Configurations
+------------------------
+
+The Gradle task ``genIntelliJRuns`` will create the Run/Debug configurations for you. Run ``./gradlew genIntelliJRuns`` 
+in the project's root directory. Perform this task **before** launching the IDE. Or, you can restart your IDE or 
+relaunch the project after you run it. You should see ``Minecraft Client`` and ``Minecraft Server`` listed as 
+application configurations and ``SpongeForge [jar]`` and ``SpongeForge [clean]`` listed as Gradle configurations.
+
+Coremods
+--------
 
 .. note::
+    Coremods do not pertain to SpongeVanilla. See the :doc:`/about/glossary` for a definition of coremod.
+
+Except for the Sponge coremod, Gradle or other automation tools provided by Sponge do not set up coremods in your 
+project. As a result, you must add them manually. To do so, specify each coremod in a comma-separated list with the 
+``-Dfml.coreMods.load`` parameter in the VM options for your project. You can locate the VM options in the Run/Debug 
+Configurations. 
+
+.. tip::
+    The **Mixins** section of :doc:`/plugin/practices/best` has a discussion on mixins, coremods, and other low-level 
+    definitions.
+
+    :doc:`mixin` also discusses other useful VM options in the **Output** section.
+
+Classpath
+---------
+
+While in the Run/Debug Configurations window, verify the ``Use classpath of module:`` is populated. Click on the 
+drop-down menu and select a module with ``_main`` in its name when it specifies ``<no module>``.
+
+.. tip:: 
+
+   When in doubt, select the project on which you are working. For example, select ``SpongeForge_main`` when working 
+   with SpongeForge or ``SpongeVanilla_main`` when working with SpongeVanilla. 
+
+Using Your IDE
+==============
+
+Now with everything working, you can do all of the usual tasks when in your IDE, such as setting breakpoints, stepping 
+through code, examining variables, and evaluating expressions. You can also utilize regular aspects of the game, such 
+as commands and log output.
+
+Debugging With Mods And Plugins
+-------------------------------
+
+When debugging code with a mod or plugin, place a copy of the mod's or plugin's jar file in the ``run/mods`` directory. 
```

Comment:
Rather than "the mod's or plugin's" can we have "it's" instead?

Reply:
Yes, a much better read for sure!
Corrected